client/asset/interface.go

```diff
@@ -41,7 +41,9 @@ type WalletConfig struct {
 	// check the confirmations on any negotiating swaps to see if action is
 	// needed. If the error is non-nil, the wallet monitoring loop encountered an
 	// error while retrieving tip information.
-	TipChange func(error)
+	TipChange         func(error)
+	MinNetworkVersion uint64
+	WalletInfo        *WalletInfo
```

Comment:
These fields are btc-clone specific, so they should be passed as parameters to `BTCCloneWallet` instead of as part of the `WalletConfig`. You might consider a separate struct `BTCCloneConfig` if the `BTCCloneWallet` parameter count is getting out of hand.